guestbook/php-redis/Dockerfile

```diff
@@ -12,7 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM php:5-apache
+FROM BASEIMAGE
```

Comment:
I see that you sed this in the Makefile. I just think it is going to be confusing for the first time user. Plus there is no reference to your Makefile anywhere.